demo/client/html/3d-demo.html.hbs

```diff
@@ -42,6 +42,12 @@
                  aria-expanded="false" aria-controls="#pf-settings">
                     <span class="pf-material-icons">settings</span>
                 </button>
+                <button id="pf-vr" type="button"
+                 class="btn btn-outline-secondary pf-toolbar-button"
+                 aria-expanded="false" aria-controls="#pf-vr"
+                 style="display: none">
+                    <span>VR</span>
```

Comment:
Note to self: we use Material Design Icons here and there is a VR icon, so let's use that.

Reply:
huh, I looked for one and didn't find it. Will look again

Reply:
https://github.com/google/material-design-icons/issues/508